static/js/admin.js

```diff
@@ -163,7 +206,49 @@ exports.setup_page = function () {
         $("#deactivation_stream_modal .stream_name").text(stream_name);
         $("#deactivation_stream_modal").modal("show");
     });
-
```

Comment:
@timabbott When I add the blank line jslint gives this error

``Fix trailing whitespace at static/js/admin.js line 209:``